src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java

```diff
@@ -1,3 +1,12 @@
+/*xml
+<module name="Checker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck">
+    <property name="max" value="80"/>
```

Comment:
It use to be 75

Reply:
Done

Reply:
The check reacts on config line which is longer than 75. What should I do in this situation? Can I move part of the text on the next line as I do with non-config content?

Reply:
See below